README.md

```diff
@@ -18,28 +18,64 @@ Then use pip to install all dependencies.
 ```bash
 pip install .
 ```
+In the main directory create `.env` file like in example below and fill
```

Comment:
This section needs to be updated:
```
After pulling the code onto your machine, start  off by creating a
`postgresql` database and puting it's credentials in 
[config.yaml](https://github.com/Czupryn013/Teaching-Platform/blob/develop/config.yaml). 
Then create a `Python Virtual Enviroment`.

python3 -m venv ".env"
source ".env/bin/activate" / source ".env/Scripts/activate"
```

Reply:
It should be placed rather in `teaching_platform`

Reply:
I think in `teaching_platform` there should be only code and all config files in main dir.